lib/semian/rails.rb

```diff
@@ -0,0 +1,7 @@
+require 'active_record/connection_adapters/abstract_adapter'
+
+class ActiveRecord::ConnectionAdapters::AbstractAdapter
```

Comment:
Could we just automagically include this from the MySQL module if `ActiveRecord` is defined? Instead of manually having to require this?